File: agent/examples/hello.js
const os = require('os');

console.log("Hello from NodeJS Agent!");
console.log(`Node Version: ${process.version}`);
console.log(`Architecture: ${os.arch()} / Platform: ${os.platform()}`);

// Small calculation
let res = 0;
for (let i = 1; i <= 100; i++) {
    res += i;
}
console.log(`Sum 1 to 100 is: ${res}`);

process.exit(0);

File: agent/examples/hello.py
import sys
import platform

print("Hello from Python Agent!")
print(f"Python Version: {platform.python_version()}")
print(f"OS Platform: {platform.platform()}")

# Try a small math operation
res = sum(range(1, 101))
print(f"Sum 1 to 100 is: {res}")

sys.exit(0)
